### DISCUSSION TITLE
Project setup for commercial license

### DISCUSSION THREAD

**Discussion by SteveGaukrodger** [emgucv/emgucv#493] [Q&A] [answered]
Hi,

I'm used to setting up Emgu from the nuget repo, but now I have a commercial version. I've downloaded and unzipped the package, but there are a lot of dlls. What exactly do I need to include in my project? There are many versions of several files and I can't find the windows runtime file - there's a project, but no dll. 

I'm running .NET Framework 4.8 (preferably, but I can upgrade if necessary).

Steve

**Comment by emgucv** [comment, marked as answer]
Thanks for purchasing the commercial license. You can find instruction to connect to the commercial nuget repository here:
https://www.emgu.com/wiki/index.php/Download_And_Installation#Commercial_Release

Using the commercial nuget repository is the easiest way to move between the open source release and the commercial release.

**Reply by SteveGaukrodger** [reply]
Thanks for the quick reply. I've followed the instructions, but clearly messed something up. I've added the repository, but still can only find the open source license. Here are my nuget settings
![image](https://user-images.githubusercontent.com/23379082/117112443-f7381900-ad80-11eb-919e-eb1f48f57bc9.png)

If I disable the other package sources, nothing is returned. Any idea what I'm doing wrong?

**Reply by SteveGaukrodger** [reply]
I figured it out - I had to change the package source in the top right corner. For some reason it was set to something other than "All". I also restarted Visual Studio, which may have helped. Either way, it's sorted now.

**Reply by emgucv** [reply]
That's great! Good to know that you got it working.
